packages/front-end/components/EventWebHooks/EventWebHookDetail/EventWebHookDetail.tsx

```diff
@@ -72,6 +135,14 @@ export const EventWebHookDetail: FC<EventWebHookDetailProps> = ({
             Edit
           </button>
 
+          <button
+            onClick={onTestWebhook}
+            className="btn btn-sm btn-outline-secondary mr-1"
+          >
+            <FaPaperPlane className="mr-1" />
+            Test
+          </button>
```

Comment:
We should probably have a loading state and disable this button.  Posting to Discord/Slack will almost always be fast, but some custom endpoints might take awhile and it's good to have UI feedback during that.

You can either implement the loading/disable part yourself or use our `<Button>` instead component which takes an async `onClick` property and will switch to a loading/disabled state while it awaits the promise automatically